store/bbolt/bbolt.go

```diff
@@ -22,9 +22,9 @@ func Open(dir string) (store.Store, error) {
 	if err != nil {
 		return nil, err
 	}
-	store := &boltStore{db: db}
-	err = store.createRootBucketIfNotExists()
-	return store, err
+	dataStore := &boltStore{db: db}
```

Comment:
maybe we can just call it `s`?

Reply:
I'm not really a fan of single letter variable names.
AFAIK it's even considered a bad practice.
Of course variable names should not become too long, but `dataStore` seems reasonable.
Any reason why you would want to rename it?